src/components/ProfileCard/ProfileCard.types.ts

```diff
@@ -0,0 +1,51 @@
+import type { CardProps } from '@mui/material/Card';
+
+export interface ProfileStatsProps {
+  /** Number of courses enrolled */
+  courses: number;
+  /** Total points earned */
+  points: number;
+  /** Current ranking position */
+  ranking: number;
+}
+
+export interface ProfileCardProps extends Omit<CardProps, 'children' | 'variant'> {
+  /** User's avatar URL */
+  avatarUrl?: string;
+  /** User's full name */
+  name: string;
+  /** User's username/handle */
+  username: string;
+  /** User's role or title */
+  role?: string;
+  /** User statistics */
+  stats?: ProfileStatsProps;
+  /** Size variant of the card */
+  size?: ProfileCardSize;
+  /** Layout variant */
+  variant?: ProfileCardVariant;
+  /** Whether to show the stats section */
+  showStats?: boolean;
+  /** Custom className */
+  className?: string;
+  /** Test id for testing purposes */
+  'data-testid'?: string;
+}
+
+export type ProfileCardSize = 'small' | 'medium' | 'large';
+export type ProfileCardVariant = 'default' | 'compact' | 'expanded';
+
+export interface StyledProfileCardProps {
+  $size: ProfileCardSize;
+  $variant: ProfileCardVariant;
+}
+
+export interface StatsItemProps {
+  icon: React.ReactNode;
```

Comment:
The `StatsItemProps` uses `React.ReactNode` but the file does not import `React` or `ReactNode`. Relying on `React` as a global type is brittle. Explicitly import the needed type to make the file self-contained.

**Sugerencia:**
```typescript
Add an import at the top:

import type { ReactNode } from 'react';

Then use `icon: ReactNode;` instead of `React.ReactNode`.
```